edge-hub/test/Microsoft.Azure.Devices.Edge.Hub.Http.Test/TwinsControllerTest.cs

```diff
@@ -163,6 +163,190 @@ public async Task TestInvokeMethodOnModuleWithResponsePayload()
             Assert.Equal(objectResult.StatusCode, (int)HttpStatusCode.OK);
         }
 
+        [Fact]
+        public async Task TestInvokeMethodWithResponsePayloadWithNoUmlautAndResponseContentLengthIsEqualToByteCount()
```

Comment:
You can add the content-length assers into the existing `TestInvokeMethod*WithResponsePayload` tests. I don't think they need to be separate tests.

Reply:
You are right.